### PR TITLE
feat: Refresh tree shortcut

### LINKED ISSUE #118: Creating a table does not reflect in the TUI
When you use the `SQL Editor` to create a table, it does not reflect inside the TUI and you have to restart lazysql to see the changes.

# Reproduction

> This was tested with SQLite

1. open `lazysql`
2. connect to a database
3. open SQL Editor and execute `CREATE TABLE` query
    - `CREATE TABLE test (id integer PRIMARY KEY);`
 4. switch back to table list -> new table is not visible

# Expectation
It would be nice to 'refresh' the table list after a new table was created, and maybe even add a shortcut for manual refreshing of the table list.

Would it be possible to filter the query-string for 'CREATE' or 'create' and if either one of them is present refresh the table list after the query was executed?

### PR DESCRIPTION
**Pull request by gytic**
This PR should fix #118 

To refresh the tree view my idea is to:
1. clear existing nodes (under root)
2. Re-initialize nodes (under root)
3. Reset search state
4. Close opened table if it no longer exists

### CODE REVIEW

**Comment by gytic** [comment]
@jorgerojas26 do you have an idea where and how to close the opened table if it was deleted?

I will look tomorrow a bit more into the code base to figure out how it is opened etc but maybe you have an idea already how one could implement that.

**Comment by jorgerojas26** [comment]
I'm not sure what you mean but maybe you just delete the tab that has the table open.

**Comment by gytic** [comment]
Okay I think this is the best I can do.

When refreshing, all nodes under the root node get deleted and afterwards re added.

File: components/Tree.go [hunk 1]
```diff
@@ -557,3 +528,48 @@ func (tree *Tree) ExpandAll() {
...
+			tree.databasesToNodes(tables, node, true)
+			App.Draw()
+		}(database, childNode)
+	}
+}
+
+func (tree *Tree) Refresh(dbName string) {
+	rootNode := tree.GetRoot()
+	rootNode.ClearChildren()
+	// readd nodes
```
Comment (ccoVeille) [inline components/Tree.go:573]:
```suggestion
	// read nodes
```

Or


```suggestion
	// re-add nodes
```

Using readd is ambiguous

Comment (gytic) [reply components/Tree.go]: this is fixed now, thanks for pointing out

**Comment by gytic** [comment]
PR is ready for review

**Comment by jorgerojas26** [comment]
This is good enough for now, i think. Can you please just change the keybinding from 'r' to 'R'. The reason is, there is a 'R' keymap for the table that refreshes the data, so, to keep consistency let's use 'R' also for the tree.

With that change, i'm ready to merge.

Thank you very much

**Comment by gytic** [comment]
I will work on it this evening

**Comment by gytic** [comment]
I ran the lints locally and had no errors. I think this should be now fixed.

Also changed the shortcut from `r` to `R`.
PR should be ready for merge now.

**Pull request merged**